templates_test/insert.tpl

```diff
@@ -41,7 +41,11 @@ func test{{$tableNamePlural}}InsertWhitelist(t *testing.T) {
 
 	tx := MustTx(boil.Begin())
 	defer tx.Rollback()
+	{{if ne .DriverName "mssql"}}
 	if err = {{$varNameSingular}}.Insert(tx, {{$varNameSingular}}Columns...); err != nil {
+	{{- else -}}
+	if err = {{$varNameSingular}}.Insert(tx, {{$varNameSingular}}ColumnsWithoutDefault...); err != nil {
```

Comment:
By default SQL Server prevents insert of explicit values for identity columns. 

If you try to perform an insert with such column, an error will be returned:

> Cannot insert explicit value for identity column in table 'Bar' when IDENTITY_INSERT is set to OFF.